lang/ir/src/ir/item/storage.rs

```diff
@@ -0,0 +1,233 @@
+// Copyright 2018-2020 Parity Technologies (UK) Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use crate::ir;
+use core::convert::TryFrom;
+use proc_macro2::Ident;
+use syn::spanned::Spanned as _;
+
+/// An ink! storage struct definition.
+///
+/// Noticed by ink! throught the `#[ink(storage)]` annotation.
+///
+/// # Note
+///
+/// An ink! smart contract must have exactly one storage definition.
+/// The storage definition must be found in the root of the ink! module.
+///
+/// # Example
+///
+/// ```
+/// # use core::convert::TryFrom;
+/// # let event = <ink_lang_ir::Storage as TryFrom<syn::ItemStruct>>::try_from(syn::parse_quote! {
+/// #[ink(storage)]
+/// pub struct MyStorage {
+///     my_value: bool,
+//      counter: u32,
```

Comment:
```suggestion
///      counter: u32,
```

Reply:
![2020-07-02-123712_1242x251_scrot](https://user-images.githubusercontent.com/8193155/86349004-c5d25680-bc60-11ea-8519-584994e36641.png)
your suggestion looks bugged to me :S

Reply:
Weird - must be the dark theme :grin: 

![image](https://user-images.githubusercontent.com/75586/86349257-ba7b2d00-bc58-11ea-89b8-1e066c430e5c.png)